### ISSUE TITLE
Where has the option to Split gone?

### ISSUE THREAD

**Issue by ljbellis** [chembl/curation-interface#111]
I have drawn a compound which I know exists in ChEMBL.

I have clicked on 'Save All To ChEMBL'

and I get this window....

There is no option to split or redraw....

Only if you click on Update do you then get the window that says this compound already exists.

This is a confusing step.
![question](https://cloud.githubusercontent.com/assets/9843928/8057221/3c1873c8-0ea7-11e5-9c61-932b48d82f51.jpg)

**Comment by mnowotka** [comment]
This is correct. If compound already exists in the database you can't make 'Split' by the definition of 'Split'. Split (as currently defined in the application logic) is: create new compound and move some records from some selected old compound to the new one. Since the compound already exists, there is no possibility to create new one so 'Split' option is not offered.

In general, after clicking 'save to ChEMBL' button, you will get two different sets of options:
1. Update | New Record | Move | Merge | Edit (Curate) | Cancel - if compound exists or:
2.  Insert new compound | Redraw | Split | Cancel - if compound doesn't exist

**Comment by mnowotka** [comment]
Because I believe that this is a correct behavior, I'm assigning this to Louisa to clarify or close this issue.

**Comment by ljbellis** [comment]
No, I think there should be an option to split for a compound which already exists.

here is the scenario:

I am looking at Compound X. It has 3 records. One of those records should be linked to Compound A. I don't know if Compound A exists, I just draw it into the window and click on Save. 

Interface tells me it exists but does not give me the opportunity to split records from another compound.

**Comment by mnowotka** [comment]
This is possible. This operation is just called MERGE. This is again 'just' about naming. I will paste a diagram, which describes how all operations are defined.

**Comment by ljbellis** [comment]
Merge is where you merge ALL the records from one compound to another

**Comment by ljbellis** [comment]
We had a long conversation about the differences with these.

**Comment by mnowotka** [comment]
This is MOVE :)

**Comment by mnowotka** [comment]
Yes, and all I say is that the functionality is there already. And no, I haven't change any names yet.

**Comment by ljbellis** [comment]
Ok, but if this is going to work for me, then the terminology needs to be relevant to me and my work.

Also, if Merge and Split do the same thing, but one is for new compounds and one is for existing, then this isn't really useful. They would need to have the same name for the same function. 

In my opinion.

**Comment by mnowotka** [comment]
Yes, I completely agree to this.

**Comment by mnowotka** [comment]
This is a document that describes current logic of the core operations offered by the curation interface:
![usecases](https://cloud.githubusercontent.com/assets/837119/9202914/055b601c-404c-11e5-885f-101b5ed0d963.png)
